src/components/product-sorting-select/product-sorting-select.module.scss

```diff
@@ -0,0 +1,31 @@
+.select {
```

Comment:
This select looks different from the select to choose product variants, which I consider has the default select styles.